### ISSUE TITLE
Undefined name property in UploadInstance

### ISSUE THREAD

**Issue by axelvaindal** [VeliovGroup/Meteor-Files#517]
Hi, 

I decided to try `MeteorFiles` in order to provide simple file upload on my dev environment.
We use `meteor@1.6` with the latest version of your package (`ostrio:files@1.9.1` for now on).

Following the guideline, I managed to declare a `FilesCollection` instance named `Images` which is available client & server side.
I also manage to call `Images.insert(config)` on client for file uploading.
However, just after the insert call, a `TypeError` is thrown in my browser console (client side) : 

```
TypeError: Cannot read property 'name' of undefined
at new UploadInstance (ostrio_files.js?hash=cb77f0e73206caa79e071cf3a24d213dc4974aa6:1557)
at FilesCollection.insert (ostrio_files.js?hash=cb77f0e73206caa79e071cf3a24d213dc4974aa6:459)
```

I have not found anything similar on your closed issues, so I'm opening this here.
Do you have some informations about this ?

Sincerely,

**Comment by axelvaindal** [comment]
Nevermind, this was a misleading of mine in my custom upload manager.
Was using files instead of file in the insert.

This being said, perhaps the Error could be : 

`You need to provide a real file to insert` when the file is missing (here was undefined).

**Comment by dr-dimitru** [comment]
@axelvaindal thank you doe suggestion.
See https://github.com/VeliovGroup/Meteor-Files/commit/2737b74ad843476e7a6e3d0be6f5d6080ea19177

**Comment by axelvaindal** [comment]
LGTM ! Thank you for updating your code so quickly 🙂

**Comment by dr-dimitru** [comment]
@axelvaindal we will publish it with next release.
Feel free to reopen it in case if the issue still persists on your end.

**Comment by dr-dimitru** [comment]
@axelvaindal thank you for contribution, sorry for delay, this fix is delivered in [v1.9.2](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.9.2), please update, let me know how it's working for you.
